doc/api/examples/map.json

```diff
@@ -5,6 +5,7 @@
         "desc": "Example map for the API",
         "permission": "commons",
         "screenshot": "https://s3.amazonaws.com/metamaps-assets/site/missing-map.png",
+        "starred": false,
```

Comment:
And by changing the schema,  i mean making the json different in different situations. I'd like the shape of the responses to be as similar as possible